src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -831,9 +831,11 @@ void Init_TestProb_Hydro_CCSN()
    Aux_Record_User_Ptr      = Record_CCSN;
    End_User_Ptr             = End_CCSN;
    Mis_GetTimeStep_User_Ptr = Mis_GetTimeStep_CCSN;
-#  ifdef TEMP_IG
-   Flu_ResetByUser_Func_Ptr = Flu_ResetByUser_Func_CCSN;
-   Flu_ResetByUser_API_Ptr  = Flu_ResetByUser_API_CCSN;
+#  if ( EOS == EOS_NUCLEAR  &&  NUC_TABLE_MODE == NUC_TABLE_MODE_TEMP )
+   if ( CCSN_Prob != Migration_Test ) {
```

Comment:
If you want to check `CCSN_Prob != Migration_Test` when `EOS_NUCLEAR` is adopted, do it in the Step (1-3) in `SetParameter()`, not here.

Reply:
I just removed this part. Now we are checking this in `Aux_Check_Parameter()`. So I think we don't need this.